interfaces/cython/SConscript

```diff
@@ -70,6 +70,9 @@ if (
 if localenv["HAS_CLANG"] and py_version_short == parse_version("3.8"):
     localenv.Append(CXXFLAGS='-Wno-deprecated-declarations')
 
+if localenv["CC"] == "icc":
+    localenv.Append(CPPDEFINES={"CYTHON_FALLTHROUGH":" __attribute__((fallthrough))"})
+
```

Comment:
Related to my previous comment, here the comparison is equality, so if someone sets `CC=/path/to/icc` this won't be picked up. This is why I'm not sure how to resolve the edge case 😄

Reply:
Me neither ... let's wait for an issue report (if it ever arises)

Reply:
Changing to `if "icc" in localenv["CC"]:` presumably takes care of the path issue, but the other one remains ... if I understand correctly, this is something that applies to *any* test for any string in an environment variable ...